versions/3.0.md

```diff
@@ -70,8 +66,8 @@ Additional utilities, such as testing tools, can also use the files.
 
 ## Definitions
 
-##### <a name="oasDefinitionFile"></a>OpenAPI Definition File
-A file or set of files which defines an API. The OpenAPI definition file uses and conforms to the OpenAPI Specification.
+##### <a name="oasDefinition"></a>OpenAPI Definition
+A file or set of files which defines an API. An OpenAPI definition uses and conforms to the OpenAPI Specification.
```

Comment:
Hmm, maybe use here "document or set of documents", as these things don't need to be in a file system at all?

(Similarly, I would replace `file` with `document everywhere where it is still left now.)